lib/solidus_dev_support/templates/extension/Gemfile

```diff
@@ -27,4 +27,4 @@ gemspec
 
 # Use a local Gemfile to include development dependencies that might not be
 # relevant for the project or for other contributors, e.g.: `gem 'pry-debug'`.
-eval_gemfile 'Gemfile-local' if File.exist? 'Gemfile-local'
+send(:eval_gemfile, 'Gemfile-local') if File.exist? 'Gemfile-local'
```

Comment:
@aldesantis can you add a comment just above referencing the issue in dependabot to make immediately clear why we're going through `send`?

Reply:
@elia all details are in the commit message, I didn't want to add too much stuff to the comments. What do you think?

Reply:
I'm fine either way, the concern was more toward the extension that will get this weird code in their gemfile, where the commit message isn't available. But it's not super important and can be merged anyway ✅

Reply:
You're right, I wasn't considering this stuff will end up in the final application. 🤦‍♂